draft-ietf-tls-tls13.md

```diff
@@ -4128,9 +4212,174 @@ Section 7 of [RFC6066], as it is not applicable to AEAD ciphers and has
 been shown to be insecure in some scenarios.
 
 
-#  Security Analysis
+#  Overview of Security Properties {#security-analysis}
+
+A complete security analysis of TLS is outside the scope of this document.
+In this section, we provide an informal description the desired properties
+as well as references to more detailed work in the research literature
+which provides more formal definitions.
+
+We cover properties of the handshake separately from those of the record layer.
+
+## Handshake {#security-handshake}
+
+The TLS handshake is an Authenticated Key Exchange (AKE) protocol which
+is intended to provide both one-way authenticated (server-only) and
+mutually authenticated (client and server) functionality. At the completion
+of the handshake, each side outputs its view on the following values:
+
+- A "session key" (the master secret) from which can be derived a set of working keys.
+- A set of cryptographic parameters (algorithms, etc.)
+- The identities of the communicating parties.
+
+We assume that the attacker has complete control of the network in
+between the parties {{RFC3552}}. Even under these conditions, the
+handshake should provide the properties listed below. Note that
+these properties are not necessarily independent, but reflect
+the protocol consumers' needs.
+
+Establishing the same session key.
+: The handshake needs to output the same session key on both sides of the
+handshake (See {{CK01}}; defn 1, part 1).
+
```

Comment:
If successful! An attacker can always disrupt the handshake, if only by cutting the link midway through. Just because one side completed successfully does not mean the other side did so.